docs/_advanced/lazy_listener.md

```diff
@@ -6,15 +6,15 @@ order: 10
 ---
 
 <div class="section-content">
-⚠️ Lazy listener functions are a beta feature to make it easier to deploy Bolt for Python apps to FaaS environments. As the feature is developed, Bolt for Python's API is subject to change.
+⚠️ Lazy listener functions are a beta feature which make it easier to deploy Bolt for Python apps to FaaS (Function-as-a-Service) environments. As the feature is developed, Bolt for Python's API is subject to change.
```

Comment:
One year ago, we decided to mark this feature as "beta" just because the feature was only available in bolt-python and we may change some of the interface if we consider porting the feature to other Bolt frameworks. 

One year has passed and we are not planning to make any breaking changes to its interface anymore. Also, I don't think it's so easy to implement the same in bolt-js due to its design limitations. As for Bolt for Java, it may be possible to implement something similar, but  as we've never received feature requests from the communities and customers, let's hold off using our time for it.

For this reason, I'd propose to remove the word "beta" from this sentence and modify it like this:

```suggestion
Lazy Listeners are a feature which make it easier to deploy Slack apps to FaaS (Function-as-a-Service) environments. Please note that this feature is only available in Bolt for Python, and we are not planning to add the same to other Bolt frameworks.
```

Reply:
@seratch 👍 Sounds good to me. I've pushed the updated language.